astropy/time/tests/test_ut1.py

```diff
@@ -72,23 +72,39 @@ def test_delta_ut1_utc(self):
             assert allclose_sec(t._delta_ut1_utc, -0.58682110003124965)
 
 
-@pytest.mark.skipif('not HAS_IERS_A')
```

Comment:
This diff is unfortunately not helpful, but I did a by-eye compare of the files for master and this branch.  I'm just trying to make sure the existing test has not been changed substantively.  You did change the way the delta and status values are computed.  I think it's equivalent, but maybe life would be easier if you just left in the original two tests `test_ut1_iers_A` and `test_ut1_iers_Auto` and then *add* tests on top?  This is finicky code so any time I see an existing test modified it rings bells.

I note that in the new `do_ut1_prediction_test` there is no check on the `jd` value being different for the non-context manager case.
```
    def test_ut1_iers_A(self):
        tnow = Time.now()
        iers_a = iers.IERS_A.open()
        tnow.delta_ut1_utc, status = iers_a.ut1_utc(tnow, return_status=True)
        assert status == iers.FROM_IERS_A_PREDICTION
        tnow_ut1_jd = tnow.ut1.jd
        assert tnow_ut1_jd != tnow.jd
```

Reply:
OK, I reorganized the file so the diff is more obvious, and re-instated the `.jd` test (I had removed it since it is possible, at least in principle, that `delta_ut1_utc` equals 0. Though probably sufficiently unlikely that one might ignore that - much more likely something is wrong when the jd doesn't change...